### ISSUE TITLE
Where is the source code for the portfolio-optimizer example?

### ISSUE THREAD

**Issue by apahl** [holoviz/panel#1388]
The [Notebook](https://github.com/holoviz/panel/blob/master/examples/gallery/demos/portfolio-optimizer.ipynb) under examples only contains the Live App URL, but no actual code.

Many thanks for your help.

Kind regards, 
Axel

**Comment by philippjfr** [comment]
All the more complex examples are hosted on examples.pyviz.org and if you click on " Download this project" at https://examples.pyviz.org/portfolio_optimizer/portfolio.html you will get an archive with the code, data and environment.

**Comment by philippjfr** [comment]
To be clear though, we need to find a better way to integrate external examples like this.

**Comment by apahl** [comment]
Thanks a lot for the link!  
I was confused because I tried the download link for the Notebook from the [gallery](https://panel.holoviz.org/gallery/demos/portfolio-optimizer.html#demos-gallery-portfolio-optimizer)

![image](https://user-images.githubusercontent.com/9406836/84036966-3a85de00-a99e-11ea-8667-effddeec1782.png)

and just got the Live App URL.

Many thanks a gain.

Kind regards,  
Axel

**Comment by np-8** [comment]
I was just about to come to ask the same question as @apahl. I was interested to see how the [Portfolio-optimizer](https://panel.holoviz.org/gallery/demos/portfolio-optimizer.html) was made, but there was only a link to a near-empty jupyter notebook. 

Could the link to the [portfolio_optimizer.zip](https://examples.pyviz.org/assets/portfolio_optimizer.zip) be added to the panel.holoviz.org gallery page for Portfolio-optimizer? Or, even a link pointing to https://examples.pyviz.org/portfolio_optimizer/portfolio.html ?

**Comment by jbednar** [comment]
I agree; I can't see any link from the Panel page to the underlying source code. I know where it is, but can't see how anyone would find it on the separate site, so I've reopened this issue.

**Comment by maximlt** [comment]
+1 on adding a link to the source code.

There's another issue with the more complex examples like this one, some of them like Glaciers have a Binder link but a file not found error is shown when binder is set up. The binder env is built from repos that are part of the `holoviz-demos` Github organization. Shouldn't the gallery just link to the running version of these notebooks deployed on *examples*?

**Comment by philippjfr** [comment]
The code now lives here: https://github.com/holoviz/panel/blob/main/examples/gallery/portfolio_optimizer.ipynb
